.github/workflows/release_validate.yml

```diff
@@ -41,37 +53,36 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Set up Dotnet
-        uses: actions/setup-dotnet@v3
-        with:
-          dotnet-version: 6.0.x
-
       - name: Check Client Go
         run: |
           mkdir -p /tmp/go && cd /tmp/go && \
           go mod init test && \
           go mod edit -require github.com/tigerbeetledb/tigerbeetle-go@v${{ inputs.version }} && \
           go mod tidy
 
-      - name: Set up Java
-        uses: actions/setup-java@v3
-        with:
-          java-version: '11'
-          distribution: 'temurin'
-
+      # Java, Node and Dotnet might take a bit to reflect.
       - name: Check Client Java
         run: |
           mkdir -p /tmp/java && \
           cat src/clients/java/validation_pom.xml | sed 's|{{VERSION}}|${{ inputs.version }}|' > /tmp/java/pom.xml && \
-          cd /tmp/java && mvn dependency:copy-dependencies && \
-          test -e target/dependency/tigerbeetle-java-${{ inputs.version }}.jar
```

Comment:
You dropped the part `test -e target/dependency/tigerbeetle-java-${{ inputs.version }}.jar` ... was that intentional?

Reply:
Yep! There was no need for it since `mvn` will exit with a non-zero exit code, and chaining it together would make the line more confusing (or require extra brackets)